### PR TITLE
Statsbox

### PR DESCRIPTION
**Pull request by gasperchan**
Stats footer stuff

### CODE REVIEW

File: css/pictures.css [hunk 1]
```diff
@@ -19,11 +19,36 @@
 }
 
 .unsorted, .pic-box {
-  height: 190px;
   overflow-y: auto;
   overflow-x: hidden;
 }
 
+.pic-box {
+  height:249px;
```
Comment (kehphin) [inline css/pictures.css:27]: wtf 249px seems like such a magic number..

Comment (gasperchan) [inline css/pictures.css:27]: it was being weird with the border lemme look into that again lol

Comment (gasperchan) [inline css/pictures.css:27]: screen shot 2015-04-09 at 10 17 26 pm

250px

screen shot 2015-04-09 at 10 17 53 pm

249 px

you can see the bottom border gets kinda wonky but i'll look into it more

**Comment by tonyjhuang** [comment]
+1

**Comment by davlin93** [comment]
+1

**Pull request merged**